includes/orphaned_objects.inc

```diff
@@ -197,16 +198,20 @@ function islandora_get_orphaned_objects() {
     foreach ($results as $result) {
       if (array_key_exists($collection_field, $result['solr_doc'])) {
         foreach ($result['solr_doc'][$collection_field] as $collection) {
-          if (in_array($collection, $missing_parents)) {
-            $orphaned_objects[] = $result;
-          }
-          elseif (!in_array($collection, $already_checked)) {
-            $test = islandora_identify_missing_parents($collection);
-            if (!$test) {
-              $orphaned_objects[] = $result;
-              $missing_parents[] = $collection;
+          if($result['PID'] != $root_pid) {
+            if ($collection != $root_pid_check) {
```

Comment:
Purely hygienic, but having a bunch of nested "if"s kinda bothers me when one could do. 

`if ( ($result['PID'] != $root_pid) && ($collection != $root_pid_check) ) {`

Seems like perhaps a less deep way to go.

Reply:
It also looks like the fact that its `if(..)` instead of `if (...)` is what is causing Travis to fail the tests so this will have to be changed regardless. From the Travis logs:

`201 | ERROR | Expected "if (...) {\n"; found "if(...) {\n"`